web/docs/project/env-vars.md

```diff
@@ -104,43 +206,70 @@ Defining environment variables in this way can be cumbersome even for a single p
 
 ## Defining Env Vars in Production
 
-While in development, we had the option of using `.env.client` and `.env.server` files which made it easy to define and manage env vars.
-However, for production, `.env.client` and `.env.server` files will be ignored, and we need to provide env vars differently.
+Defining env variables in production will depend on where you are deploying your Wasp project. In general, you will define them via mechanisms that your hosting provider provides.
+
+We'll talk about how to define env vars for each deployment option in the [deployment section](../deployment/env-vars.md).
+
+## Custom Env Var Validations
+
+TODO: when the Zod validation PRs are merged, describe how users can define their own validations.
 
-![Env vars usage in development and production](/img/env/prod_dev_fade_2.svg)
```

Comment:
What happened to this image?

Reply:
It moved to the deployment env vars page :)